stage1/init/init.go

```diff
@@ -627,7 +627,12 @@ func stage1() int {
 		log.PrintE("continuing with per-app isolators disabled", err)
 	}
 
-	if err = stage1common.WritePpid(os.Getpid()); err != nil {
+	pid_filename := "ppid"
+	if flavor == "kvm" {
+		pid_filename = "pid"
+	}
```

Comment:
I'd add a comment explaining why it is pid or ppid, and maybe a link to the stage1 doc.